src/index.html

```diff
@@ -2,16 +2,193 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="inputForm"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldsetBlock">
+        <legend>Personal information</legend>
+        <div class="inputBlock">
+          <label for="surname">Surname:
+            <input
```

Comment:
let's put a text on a new line to improve readability, fix it everywhere
```suggestion
          <label for="surname">
            Surname:
            <input
```